astro/src/content/docs/lifecycle/manage-users/user-actions.mdx

```diff
@@ -52,7 +51,7 @@ If a temporal Action instance has ended, it is still considered active. Active r
 
 Below is a visual reminder of the relationships between [Tenants&#44; Groups&#44; and Applications](/docs/get-started/core-concepts/).
 
-<Diagram active="User" />
+<img src="/img/docs/get-started/core-concepts/core-concepts-relationships-users.png" alt="Diagram showing Users used within FusionAuth" />
```

Comment:
There are still 2 diagrams on this page that are in mermaid.  Is that OK?

Reply:
@mark-robustelli Yes these will get done later, each milestone in the project is for different types.